src/Command/AbstractCommand.php

```diff
@@ -48,15 +46,15 @@
  * $users = $connectionInterface->createCommand('SELECT * FROM user')->queryAll();
  * ```
  *
- * Command supports SQL statement preparation and parameter binding.
+ * Supports SQL statement preparation and parameter binding.
```

Comment:
I think it was OK before. Here we don't have the context anymore. Last time we mentioned an object it was SQL statement so it becomes confusing.